helm-chart/values.yaml

```diff
@@ -41,9 +43,21 @@ securityContext: {}
   # runAsNonRoot: true
   # runAsUser: 1000
 
+pvc:
+  storageClassName: standard
+  conf:
+    size: 1Gi
+  data:
+    size: 1Gi
+  env:
+    size: 1Gi
+  log:
+    size: 1Gi
```

Comment:
Same here, this might be too small

Reply:
It's just a default value, but I can increase it if you want :)

Reply:
Okay, I have no experience with helm, but what if a user starts their iTop with this default and the logs become this size? Will that be blocking (disk full)? Will they be able to resize easily?

Reply:
Yes, when you want deploy itop with helm chart you'll have to set some option, ingress host addr, bdd creds etc .. 
And now you can also overwrite all values present in values.yaml. 

And if you install with 1Gi volumes, you can at any moment upgrade size of volume (please see kube [doc](https://kubernetes.io/docs/concepts/storage/storage-classes/#allow-volume-expansion)

Reply:
Okay thanks, its is fine for me in that case.